### PR TITLE
Adds new lines to breakdown speech list

### PR DESCRIPTION
**Pull request by Humonitarian**
## About The Pull Request

Adds a few new lines to breakdown list. Some are references, some are sourced locally.

## Why It's Good For The Game

More variety good.

## Testing

Saw no problems launching local server

## Changelog
:cl:
add: new insanity lines
/:cl:

### CODE REVIEW

**Comment by Humonitarian** [comment]
@Firefox13

**Comment by null-Aurelian** [comment]
"You're just like me, trying to make history"
"Get it off me! GET IT OFF ME!"

**Comment by Mycah142** [comment]
";Soup."

**Review by TheShown911 — APPROVED** [review]
5 is the number

File: code/modules/sanity/sanity_lists.dm [hunk 1]
```diff
@@ -81,7 +81,13 @@
 		"But they know, they always know. Accept this.",
 		"Something inside your skull cracks. You feel no pain.",
 		"Curiosity is your greatest sin.",
-		"A burger would be good. Anything edible, really."
+		"A burger would be good. Anything edible, really.",
+		"What can possibly be gained by upsetting the current order?",
+		"You have chosen your answer. Now see it through.",
+		"Noone cared who I was until I put on the scarf.",
```
Comment (hyperioo) [inline code/modules/sanity/sanity_lists.dm:87]:
```suggestion
		"No one cared who I was until I put on the scarf.",
```

File: code/modules/sanity/sanity_lists.dm [hunk 2]
```diff
@@ -81,7 +81,13 @@
...
+		"Pills. Vape. Inject.",
+		"The snow. It calls to me... but the warm holds me tightly.",
```
Comment (hyperioo) [inline code/modules/sanity/sanity_lists.dm:89]:
```suggestion
		"The cold. It calls to me... but the warm holds me tightly.",
```
Sounds more ominous. And we don't have snow on Eris so it's kinda weird to blurt that out of nowhere.

File: code/modules/sanity/sanity_lists.dm [hunk 3]
```diff
@@ -169,7 +175,16 @@
 			"The rules are written in blood. The very old blood of generations of idiots.",
 			"You are coming home.",
 			"The writings are here, behind your skull.",
-			"YOU HAVE NO CHOICE"
+			"YOU HAVE NO CHOICE",
+			"Everything is twisted... you... this world... all of it.",
+			"Welcome to the Abyss.",
+			"Human arrogance will eventually usher in our downfall.",
+			"Millions will die. Exciting, don't you think?",
+			"The price we all must pay will be great; atonement shall not be easy. The blame lies with all of us.",
```
Comment (hyperioo) [inline code/modules/sanity/sanity_lists.dm:183]:
```suggestion
			"The price we all must pay will be great; atonement shall not be easy. The blame lies on all of us.",
```
"Lie with" is more "to have sexual relations with"

**Review by hyperioo — APPROVED** [review]

File: code/modules/sanity/sanity_lists.dm (hunk 3 above)
Comment (Humonitarian) [reply code/modules/sanity/sanity_lists.dm:183]: Then it should be rests on all of us, no?

iirc the blame lies with you specifically as a reference to sleeping with the blame

File: code/modules/sanity/sanity_lists.dm (hunk 2 above)
Comment (Humonitarian) [reply code/modules/sanity/sanity_lists.dm:89]: Moebius Chemistry disagrees

File: code/modules/sanity/sanity_lists.dm (hunk 3 above)
Comment (hyperioo) [reply code/modules/sanity/sanity_lists.dm:183]: I was refering myself to the wiktionary
> Used with [with](https://en.wiktionary.org/wiki/with#English): to have sexual relations with.
> _And it came to passe after these things, that his masters wife cast her eyes vpon Ioseph, and shee said, Lie with me._
>
>Used with [on](https://en.wiktionary.org/wiki/on#English)/[upon](https://en.wiktionary.org/wiki/upon#English): to [be](https://en.wiktionary.org/wiki/be) [incumbent](https://en.wiktionary.org/wiki/incumbent) (on); to be the responsibility of a person.

But it's true that on a figurative sense, lie with could work.

Comment (Humonitarian) [reply code/modules/sanity/sanity_lists.dm:183]: Yeah you lie with the guilt the same way you lie with the devil, it specifically refers to bedding these concepts

**Pull request merged**